packages/react-devtools-scheduling-profiler/src/view-base/Surface.js

```diff
@@ -107,12 +118,32 @@ export class Surface {
   }
 
   handleInteraction(interaction: Interaction) {
-    if (!this.rootView) {
-      return;
+    const rootView = this.rootView;
+    if (rootView != null) {
+      const viewRefs = this._viewRefs;
+      switch (interaction.type) {
+        case 'mousemove':
+          // Clean out the hovered view before processing a new mouse move interaction.
```

Comment:
just curious, why didn't we need to do this before?

Reply:
TBH I'm just kind of tweaking bits of the UX code like this as I go b'c it's something I've never built before. (Usually I just rely on e.g. browsers to handle these things). Probably will need more tweaking and refactoring over time as we go and test it out more.